### PR TITLE
added details into the README

### PR DESCRIPTION
**Pull request by laUmar123**

### CODE REVIEW

File: pipeline/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+| **transform.py** | Once the data has been fetched, all unnecessary data is removed and validated. By the end of the transform process, only data with valid values is kept. | 
+| **load.py** | This file is responsible for loading data into the database. This includes both earthquake data and new data that wasn't previously in the database such as earthquake monitoring stations. |
+| **main.py** | This file contains the entire pipeline process, i.e. running extract, transform and load with the correct arguments. | 
+| **handler.py** | Provides the same service as 'main.py' with the only difference being the structure of the file - this file is going to be used as the Lambda function. |
+| **test_*.py** | Files starting with 'test_' are used for testing each step of the pipeline. |
+| **\*.tf** | Files ending in '.tf' are used to terraform related AWS services, such as the EventBridge scheduler used to run the pipeline every minute. |
+| **Dockerfile** | Used to dockerise the pipeline. |
+| **requirements.txt** | A list of all the required modules needed to run the pipeline. |
```
Comment (fm1psy) [inline pipeline/README.md:14]: I like the use of a table, good job

File: pipeline/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+## Instructions for running pipeline
+1. Create a virtual environment and download all required modules - assuming you are in the pipeline folder:
+    ```
+    python3 -m venv .venv
+    source .venv/bin/activate
+    pip install -r requirements.txt
+    ```
+2. Setup the following environmental variables in a `.env` file:
+    | Variable Name | Value |
+    | ------------- | ----- |
```
Comment (fm1psy) [inline pipeline/README.md:24]: I believe you are missing db name from the table.

**Review by fm1psy — CHANGES_REQUESTED** [review]
There seems to be an element missing from this document. Please go back and look over my comments.

File: pipeline/README.md (hunk 2 above)
Comment (laUmar123) [reply pipeline/README.md:24]: Apologies for that, going to add it now.

**Review by fm1psy — APPROVED** [review]

**Pull request merged**